### PR TITLE
Kishorebolinidi home town

### PR DESCRIPTION
**Pull request by ananthuluva**
Test for pull request

### CODE REVIEW

**Comment by kishorebolinidi** [comment]
Testing email 123... :dancer:

File: vijayawada_ap_india.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Guru Nanak Colony Road, Benz Circle
+Opens at 11:00 AM
+
+- [ ] KFC
+4.0 (125) · Chicken
+Fast-food chain known for its buckets of fried chicken, plus wings & sides.
+D. No. 42-1-52D & 52E, Upper Ground Floor, K K Tower, MG Road
+Opens at 11:00 AM
+
+- [ ] Babai Hotel
```
Comment (rvgidda) [inline vijayawada_ap_india.md:22]: Oh wow!!

**Comment by rvgidda** [comment]
Hello Mohan

**Comment by rakinepalli** [comment]
:+1:

**Comment by kishorebolinidi** [comment]
Hola :accept:

**Pull request merged**